docs/whatsnew/0.1.0.rst

```diff
@@ -34,8 +34,7 @@ New Features
   <http://docs.plasmapy.org>`_ that is hosted by `Read the Docs
   <https://readthedocs.org/>`_.
 
-  - Automated documentation builds with `Sphinx
-    <http://www.sphinx-doc.org/>`_.
+  - Automated documentation builds with Sphinx.
```

Comment:
How about adding

```rst
.. _Sphinx: https://www.sphinx-doc.org
```

to `docs/common_links.rst` so it can be used in places like here.